### ISSUE TITLE
Is this for dds238 meter?

### ISSUE THREAD

**Issue by ccfiel** [deontgn/dds238-2zn#1]
Hello,
Is this for dds238-4 W meter?

![meter](https://user-images.githubusercontent.com/1788360/53401032-ff021300-39e9-11e9-8117-7e2c198f3351.jpg)

**Comment by deontgn** [comment]
Hi, this is for DDS238-2 ZN/S. 
![image](https://user-images.githubusercontent.com/1673669/53410420-d6235300-39d4-11e9-9615-099aceca1b3d.png)

**Comment by ccfiel** [comment]
thanks! what are you using to communicate rs485 to esp8266? is it RS485 Adapter 485 to UART Serial ?

![scm-ttl-to-rs485-converter-485-to-uart-serial-port-automatic-flow-control-module](https://user-images.githubusercontent.com/1788360/53456721-59db4f00-3a6a-11e9-8655-512962950049.jpg)

**Comment by deontgn** [comment]
No, i use RS485 MAX485CSA -> arduino uno (ATmega328P) -> nodemcu (esp8266)

**Comment by ccfiel** [comment]
Why did you put arduino uno. Would it be possible just RS485 MAX485CSA  ->
nodemcu (esp8266)?
